integration-test/run.sh

```diff
@@ -148,15 +148,22 @@ npm install $DIST/serverless-haskell-*.tgz
 assert_success "sls package" sls package
 
 # Test local invocation
-assert_expected_output "sls invoke local" local_output.txt \
+assert_contains_output "sls invoke local: logs" "This should go to logs" \
+    sls invoke local --function main --data '[4, 5, 6]'
```

Comment:
Do you really need to run `sls invoke local` multiple times? Why not just run it once with output redirected to a test file and then check that it contains all the expected strings?

Reply:
Yes, it would be better; however, as far as I can see it's not even recompiled so I think it's acceptable. This whole bash script can probably be rewritten in a proper test framework with assertions, though.

Reply:
Ok